scripts/utils.js

```diff
@@ -39,6 +39,8 @@ const promisifySpawn = (...args) => new Promise((res, rej) => {
   const spawnedProcess = spawn(...args);
   let stdout = '';
   let stderr = '';
+  spawnedProcess.stdout.setEncoding('utf8');
+  spawnedProcess.stderr.setEncoding('utf8');
```

Comment:
Is this an enhancement or required with the upgrade ?

Reply:
this is actually left over from some changes to log on `data` instead of at the end. it makes the chunks strings instead of buffers. it can be removed